docs/components/grid.md

```diff
@@ -0,0 +1,128 @@
+<script setup lang="ts">
+import SGrid from 'sefirot/components/SGrid.vue'
+import SGridItem from 'sefirot/components/SGridItem.vue'
+</script>
+
+# SGrid
+
+`<SGrid>` is a utility component to handle CSS grid layout.
+
+<Showcase
+  path="/components/SGrid.vue"
+  story="/stories-components-sgrid-01-playground-story-vue"
+>
+  <SGrid cols="4" gap="24">
+    <SGridItem>
+      <div class="h-64 rounded-6 bg-info" />
+    </SGridItem>
+    <SGridItem>
+      <div class="h-64 rounded-6 bg-info" />
+    </SGridItem>
+    <SGridItem>
+      <div class="h-64 rounded-6 bg-info" />
+    </SGridItem>
+    <SGridItem>
+      <div class="h-64 rounded-6 bg-info" />
+    </SGridItem>
+    <SGridItem>
+      <div class="h-64 rounded-6 bg-info" />
+    </SGridItem>
+    <SGridItem>
+      <div class="h-64 rounded-6 bg-info" />
+    </SGridItem>
+    <SGridItem>
+      <div class="h-64 rounded-6 bg-info" />
+    </SGridItem>
+    <SGridItem>
+      <div class="h-64 rounded-6 bg-info" />
+    </SGridItem>
+  </SGrid>
+</Showcase>
+
+## Usage
+
+Use `<SGrid>` and `<SGridItem>` component to construct the grid structure.
+
+```vue
+<script setup lang="ts">
+import SGrid from '@globalbrain/sefirot/lib/components/SGrid.vue'
+import SGridItem from '@globalbrain/sefirot/lib/components/SGridItem.vue'
+</script>
+
+<template>
+  <SGrid cols="4">
+    <SGridItem span="2">
+      <div>...</div>
+    </SGridItem>
+    <SGridItem span="1">
+      <div>...</div>
+    </SGridItem>
+  </SGrid>
+</template>
+```
+
+You have the flexibility to define the `:cols` and `:gap` props when using the `<SGrid>` component, allowing you to have full control over the grid layout.
```

Comment:
We don't label props with colons in the docs, should we remove here too to keep consistency?

```suggestion
You have the flexibility to define the `cols` and `gap` props when using the `<SGrid>` component, allowing you to have full control over the grid layout.
```

Reply:
I've noticed there is a mixture of these existing in most recent PR's. Are you implying props should always be prefixed with a colon or is there some sort of context to why some props are prefixed and others are not throughout the docs?

Reply:
Ah, yeah... maybe. What do you think 🤔 Still not 100% sure. But maybe I'm thinking it could be easier to read if we follow these rules? 👀 

- `<SComponent>`
- `:prop`
- `@event`

Reply:
> * `<SComponent>`
> * `:prop`
> * `@event`

Agree, this makes sense and gives connotations more clarity. Thanks for confirming!

Reply:
👍